CHANGELOG.md

```diff
@@ -1,19 +1,22 @@
-### Unreleased
+# Changelog
+This changelog keeps track of all changes to the packs-sdk. We follow conventions from [keepachangelog](https://keepachangelog.com/en/1.0.0/).
+
+## [Unreleased]
 - **Breaking Change** ValueHintType.Url will now create a column of type "Link" instead of "Text".
 - Added ValueHintType.Email for new column type "Email".
 
-### 0.8.2
+## [0.8.2]
```

Comment:
for some reason we need `[]` surrounding the version number and they need to all be `##` headers

Reply:
do you need [] for the version numbers below, or just the first one?

Reply:
for some reason just the first one since it only validates for your "current version" when you do run the `release-it` command. I can fix up the inconsistency in a follow-up tho